src/VirtualMonoRepo/allowed-binaries.txt

```diff
@@ -0,0 +1,60 @@
+*.bmp
+*.docx
+*.gif
+*.ico
+*.jpg
+*.JPG
+*.pdf
+*.png
+*.PNG
+*.rtf
+*.snk
+*.vsd
+*.vsdx
+*.xlsx
+
+**/test/*
+**/Test/*
+**/Test/*
+**/testCert*.pfx
+**/TestCert*.pfx
+**/tests/*
+
+src/arcade/src/Microsoft.DotNet.SignTool.Tests/*
+
+src/aspnetcore/**/samples/*
+src/aspnetcore/**/TestCertificates/*
+src/aspnetcore/src/*.eot
+src/aspnetcore/src/*.otf
+src/aspnetcore/src/*.ttf
+src/aspnetcore/src/*.woff
+src/aspnetcore/src/*.woff2
+src/aspnetcore/src/Components/Web.JS/dist/Release/blazor.server.js # JavaScript file with a null byte
+src/aspnetcore/src/Components/Web.JS/dist/Release/blazor.webview.js # JavaScript file with a null byte
+src/aspnetcore/src/ProjectTemplates/Web.ProjectTemplates/**/app.db
+src/aspnetcore/src/submodules/spa-templates/**/app.db
+
+src/fsharp/**/signedtests/*
+src/fsharp/src/fsi/fsi.res # Icon
+src/fsharp/tests/fsharp/core/resources/*.wav
+
+src/msbuild/src/Tasks.UnitTests/**/*.bin
+src/msbuild/src/Tasks.UnitTests/**/CacheFileSamples/*.cache
+
+src/razor/**/SampleApp/**/fonts/*
+
+src/roslyn/**/CodeAnalysisTest/*
+
+src/runtime/src/libraries/System.Diagnostics.EventLog/src/Messages/EventLogMessages.res # Icon
+src/runtime/src/mono/mono/eglib/test/*.txt # Some non UTF8 text files
```

Comment:
I see a couple exclusions with test folders.  These should be cleaned up now that we exclude test folders.